src/components/PrivacyPolicy/PrivacyTerms.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+/* import React from "react";
```

Comment:
Is it ok we don't have this component anymore?

Reply:
Yes, it was referring to these two translation keys: inputCardPage.privacyTerms, inputCardPage.privacyDesc which no longer exist